modules/contacts/contacts.service.js

```diff
@@ -0,0 +1,67 @@
+const Contact = require('./contacts.model');
+const {
+  CONTACT_NOT_FOUND,
+  CONTACT_ALREADY_EXIST,
+} = require('../../error-messages/contacts.messages');
+
+class ContactsService {
+  async getContacts() {
+    return await Contact.find();
+  }
+
+  async getContactById(id) {
+    const contact = await Contact.findById(id);
+
+    if (contact) {
+      throw new Error(CONTACT_NOT_FOUND);
+    }
+
+    return contact;
+  }
+
+  async addContact(data) {
+    if (await this.checkContactExist(data)) {
+      throw new Error(CONTACT_ALREADY_EXIST);
+    }
+
+    return new Contact(data).save();
+  }
+
+  async updateContact(id, contactData) {
+    const contact = await Contact.findByIdAndUpdate(id, contactData);
+
+    if (!contact) {
+      throw new Error(CONTACT_NOT_FOUND);
+    }
+
+    return contact;
+  }
+
+  async deleteContact(id) {
+    const contact = await Contact.findByIdAndDelete(id);
+
+    if (contact) {
+      return contact;
+    }
+
+    throw new Error(CONTACT_NOT_FOUND);
+  }
```

Comment:
Change it as you did in previous methods. 
if (!contact) {
throw new Error(CONTACT_NOT_FOUND);      
    }
return contact;